cumulus/pallets/xcmp-queue/src/bridging.rs

```diff
@@ -63,13 +63,6 @@ impl<SiblingBridgeHubParaId: Get<ParaId>, Runtime: crate::Config>
 			return true
 		}
 
-		// TODO: https://github.com/paritytech/polkadot-sdk/pull/1556 - once this PR is merged, we may
-		// remove the following code.
-		// TODO: the following restriction is arguable, we may live without that, assuming that
-		// There can't be more than some `N` messages queued at the bridge queue (at the source BH)
-		// AND before accepting next (or next-after-next) delivery transaction, we'll receive the
-		// suspension signal from the target parachain and stop accepting delivery transactions.
```

Comment:
I guess we could keep these lines actually, wdyt @svyatonik ?

Reply:
We may or may not leave it. Like this "arguable" may also sound confusing to readers. What remains is enough to justify the code imo